src/models/delta.coffee

```diff
@@ -140,11 +141,10 @@ class DeltaStream extends EventEmitter
     @timeoutId = setTimeout @restartBackoff.backoff.bind(@restartBackoff), Delta.streamingTimeoutMs
 
   _onError: (err) ->
-    console.error 'Nylas DeltaStream error:', err
-    @restartBackoff.reset()
     @emit('error', err)
+    @restartBackoff.reset()
 
   _restartConnection: (n) ->
-    console.log "Restarting Nylas DeltaStream connection (attempt #{n + 1}):", @request?.href
```

Comment:
some people might be relying on this, i don't think it's a good idea to remove it.

Reply:
Relying on a `console.log`?

Reply:
yes - some people might might be monitoring the logs and relying on this to see if anything goes wrong with the delta stream. 
Not sure if this is a blocker though.

Reply:
You're right, that might happen.

Reply:
regardless @kewitz you might need to call grunt to compile your changes into the /lib